unskript-ctl/unskript_ctl.py

```diff
@@ -29,8 +29,6 @@
 from argparse import ArgumentParser, REMAINDER, SUPPRESS
 from db_utils import *
```

Comment:
This file is removed from unskript_ctl As it is being replaced with unskritp_ctl_main.py